cpu/sam0_common/periph/flashpage.c

```diff
@@ -103,7 +103,15 @@ void flashpage_write(int page, const void *data)
 
     /* write data to page */
     if (data != NULL) {
-        flashpage_write_raw(page_addr, data, FLASHPAGE_SIZE);
+      /* One RIOT page is 4 SAM0 flash pages (a row) as described in the file
```

Comment:
The docs are nice, but I wonder if the number "4" can be derived from other values, and if you can provide some sort of check in case any value changes.

Reply:
about removing that "4": I would give that a look in depth asap, but should we tackle it as a separate PR since it's something that was there already before (to make that change more traceable and not mixed into this one)?